bundles/AdminBundle/Controller/Admin/SettingsController.php

```diff
@@ -326,7 +327,7 @@ public function getSystemAction(Request $request)
     {
         $this->checkPermission('system_settings');
 
-        $values = Config::getSystemConfig();
+        $values = Config::getSystemConfiguration();
```

Comment:
I would also remove internal usages of Config and get the parameters directly from the Kernel. I'd still leave the Config class for BC reasons though, but it should trigger a deprecation notice for `getSystemConfig`